specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/GetOperations.json

```diff
@@ -0,0 +1,49 @@
+{
+  "parameters": {
+    "api-version": "2021-03-15"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "Microsoft.Quota/quotaLimits/read",
+            "display": {
+              "provider": "Microsoft Quota",
+              "resource": "Resource service limit",
+              "operation": "Get resource service limit",
+              "description": "Get the current service limit or quota of the specified resource and location"
+            }
+          },
+          {
+            "name": "Microsoft.Quota/quotaLimits/write",
+            "display": {
+              "provider": "Microsoft Quota",
+              "resource": "Resource service limit",
+              "operation": "Create resource service limit",
+              "description": "Create service limit or quota for the specified resource and location"
+            }
+          },
+          {
+            "name": "Microsoft.Quota/quotaLimitProviders/read",
+            "display": {
+              "provider": "Microsoft Quota",
+              "resource": "Quota limits provider metadata",
+              "operation": "Get quota limit providers metadata",
+              "description": "Get quota limit resource providers metadata"
+            }
+          },
+          {
+            "name": "Microsoft.Quota/resourceProviders/locations/quotaLimitsRequests/read",
```

Comment:
>Microsoft.Quota/resourceProviders/locations/quotaLimitsRequests/read [](http://example.com/codeflow?start=21&length=68)

It should be 'Microsoft.Quota/quotaLimitsRequests/read' in the current model.

Reply:
Will update it.